man/sd_notify.xml

```diff
@@ -251,6 +251,13 @@
         submitted name does not follow these restrictions, it is ignored.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>FDPOLL=0</term>
+
+        <listitem><para>When used in combination with <varname>FDSTORE=1</varname>, it disables polling of the stored
+        file descriptors regardless of whether or not they are pollable.</para></listitem>
```

Comment:
hmm, would like to see a brief discussion for the why and why not here. i.e. just a sentence: "it's generally recommended not to use this option, as file descriptors referring to objects that cease to exist should ideally be cleaned up right away automatically; use this only in cases where the file descriptor should remain alive beyond that point, for example to schedule precisely the time when it is close", or something like that...

Reply:
I have added some more information and noted that its use is not generally recommended.